.github/workflows/build.yml

```diff
@@ -64,37 +54,43 @@ jobs:
       build-version: ${{ steps.generate-build-version.outputs.build-version }}
     steps:
       - name: Checkout code
-        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # ratchet:actions/checkout@v4
+        uses: actions/checkout@v4
         with:
           ref: ${{ (inputs.branch != '' && inputs.branch) || '' }}
           fetch-depth: 0
+
       - name: Sette yarn-config
         run: |
           yarn config set npmScopes.navikt.npmRegistryServer "https://npm.pkg.github.com"
           yarn config set npmScopes.navikt.npmAlwaysAuth true
           yarn config set npmScopes.navikt.npmAuthToken $NPM_AUTH_TOKEN
         env:
           NPM_AUTH_TOKEN: ${{ secrets.READER_TOKEN }}
-      - name: Set up NODE med yarn
-        uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # ratchet:actions/setup-node@v3
+
+      - uses: actions/setup-node@v4
         with:
-          node-version: ${{ inputs.node-version }}
+          node-version: 22.17.1
           cache: 'yarn'
+
       - name: Use turbo cache
-        uses: actions/cache@v4
+        id: hent-cache
+        uses: actions/cache/restore@v4
         with:
           path: ~/work/fp-frontend/fp-frontend/.turbo
           key: turbo-${{ runner.os }}-${{ github.ref_name }}-${{ hashFiles('yarn.lock') }}-${{ inputs.app }}
-          restore-keys: turbo-${{ runner.os }}-master-${{ hashFiles('yarn.lock') }}-${{ inputs.app }}
+          restore-keys: |
+            turbo-${{ runner.os }}-${{ github.ref_name }}-${{ hashFiles('yarn.lock') }}-${{ inputs.app }} |
+            turbo-${{ runner.os }}-master-${{ hashFiles('yarn.lock') }}-${{ inputs.app }}
+
       - name: Installere dependencies
         run: yarn install --immutable
-      - name: Build and linting
+
+      - name: Build
```

Comment:
Om ein tek eslint og stylelint ut ifrå dependsOn på build så syns eg ein bør køyra desse før build her da det ikkje er noko poeng i å bygga om desse feilar (og eslint/stylelint er kjappare å køyra)

Reply:
Det er flere grunner til at jeg dro lintingen ut av dependsOn og inn i egen workflowen. Den ene grunnen er fordi det øker tiden. 
Den andre er at om lintingen feiler så synes jeg den ikke bør blokkere kjøring av test og bygg. Dette er spesielt mtp. at vi bør legge opp til en pipeline som tillater oss å prodsette raskt ved behov. 
Tredje grunnen er at det er raskere feedback til utvikler. For hvis man kan få beskjed om at både linting og test har feilet på en og samme gang så sparer i hvertfall jeg meg tid på å fikse to fluer i en smekk 😅

Reply:
Las dette litt kjapt, eg trudde lint var køyrt i hovedflyten fortsatt, berre etter build. Ja er einig i det du seier, bra jobba!